heroku/cli
# refactor: migrate members:add to oclif/core

[Work item](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001awRKnYAM/view)

## Testing
- Check out this branch and run `yarn` and `yarn build`
- Pick a team to test on (I like to use hk-front-end-online-test-team and hk-front-end-legacy-ent-team)
- Adding a member to a team
    - Do not use hk-front-end-online-test-team for this. It is configured to send invites.
    - The role can be set to admin, member, collaborator, or owner
    - Add a member to a team using `./bin/run members:add -t TEAM-NAME -r ROLE EMAIL
- Inviting a member to a team
    - hk-front-end-online-test-team is configured for this
    - Invite a member to the team using `./bin/run members:add -t hk-front-end-online-test-team -r ROLE EMAIL`
